lib/numbers.md

```diff
@@ -5,20 +5,25 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+The main types of numbers in Ruby are Fixnums (integers), and Floats (decimals). Numbers can also be Strings in Ruby if indicated by quotation marks.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+You can use all the basic math operators to perform operations on numbers, including addition, substraction, division, multiplication, and so forth. 
+
+You can also determine if a number is less than, greater than, or equal to another number.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The + operator in Ruby can be used to concatenate Strings, or combine Strings together into a single String. So you could use + to create the String "I love elephants" by putting: puts "I" + "love" + "elephants".
```

Comment:
A minor correction: The code `"I" + "love" + "elephants"` would not create the string `"I love elephants"`. It would create the string `"Iloveelephants"`; because spaces are characters too, and there are no spaces in any of the strings being concatenated :)